src/components/views/spaces/SpaceTreeLevel.tsx

```diff
@@ -68,8 +69,15 @@ export class SpaceItem extends React.PureComponent<IItemProps, IItemState> {
     constructor(props) {
         super(props);
 
+        const collapsed = SpaceTreeLevelLayoutStore.instance.getSpaceCollapsedState(
+            props.space.roomId,
+            this.props.parents,
+            !props.isNested,
+        );
+
         this.state = {
-            collapsed: !props.isNested, // default to collapsed for root items
+            // default to collapsed for root items
```

Comment:
this comment doesn't make much sense anymore given we're not doing anything to the variable ourselves. I'd suggest moving it inline to Line 75 where we set the default.